src/ecrc-frontend/src/components/page/transition/Transition.js

```diff
@@ -20,10 +20,15 @@ export default function Transition({
       <div className="page">
         <div className="transition-content col-md-8">
           <p>
-            You are being redirected to the legacy online criminal record check
-            (eCRC) application. You will be required to re-enter your access
-            code to continue. If you are not redirected automatically, please
-            click <a href="https://justice.gov.bc.ca/eCRC/home.htm">here</a>
+            You are unable to login with your BC Services Card at this time.
```

Comment:
Should this be a part of the default text or in the bcsc section? All users will be shown this

Reply:
the text "you are unable to login....." should be shown to all users I believe

Reply:
Kk, just wanted to make sure. Looks confusing to be told you are unable to login when all you had done is put an org code in.